backend/pom.xml

```diff
@@ -51,9 +57,9 @@
     </dependency>
     
     <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <version>3.3.3</version>
```

Comment:
Are there no tests using mockito at the moment? If so it's ok to remove this for now.

Reply:
I did remove it before. Is it showing up on your end? On my end, I see those lines as removed.

Reply:
I seem them as removed by this PR. I'm saying it's ok to do that now but it will almost certainly be added back in the near-future as more backend tests are added.

Reply:
Oh, I get it. I don't have any mockito tests right now so I removed the dependency.